src/main/webapp/WEB-INF/tags/instructor/results/questionPanel.tag

```diff
@@ -25,7 +25,7 @@
                 <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_RESULTS_GROUPBYSECTION%>" value="${data.selectedSection}">
             </form>
             <div class='display-icon pull-right'>
-                <span class="glyphicon ${ isShowingResponses ? 'glyphicon-chevron-up' : 'glyphicon-chevron-down'} pull-right"></span>
```

Comment:
@boxin-yang based on the experiments i did i think `isShowingResponses` is set true when the response is showing. The values up and down were exchanged by mistake earlier so it was showing the reverse values. So if i just reverse it works fine. I have changed it and it's working fine. Please have a look at it.